Misc/NEWS.d/next/Library/2021-08-04-12-29-00.bpo-44822.zePNXA.rst

```diff
@@ -0,0 +1,3 @@
+:mod:`sqlite3` now raises :exc:`OverflowError` if an user-defined function
```

Comment:
Actually sqlite3 overwrites OverflowError with other exception. But it does not matter, the most important part of this PR is that result strings with NUL are now correctly supported.

Something like (please correct my English):
```suggestion
Fix support of user functions and aggregates returning string containing NUL in :mod:`sqlite3`.
```